src/Kerbero.Infrastructure/NukiActions/Repositories/NukiSmartLockExternalRepository.cs

```diff
@@ -0,0 +1,95 @@
+using System.Net;
```

Comment:
Create a new interface
```csharp
public interface ITokenAuthenticationHolder
{
	public string? Token { get; set; }

	public void SetToken(string? token);
}
```
This allow us to propagate the concept of "something that has an authentication token" to other classes.

Also the interactor should be updated with something like this

```csharp
(_nukiSmartLockExternalRepository as ITokenAuthenticationHolder)?.SetToken(account.Value.Token);
```